packages/ui/src/components/primitives/ServerCard.tsx

```diff
@@ -0,0 +1,99 @@
+import type { ServerPublic } from '@answeroverflow/api';
+import Link from 'next/link';
+import { Button, LinkButton, LockIcon } from './base';
+import { ServerIcon } from './ServerIcon';
+import Image from 'next/image';
+
+export type ManageServerCardProps =
```

Comment:
Not a huge fan of this interface, this whole file needs some refactoring

ServerCard needs to take in react components for the description section, button section, and image section instead that will let us be more flexible